neuralmonkey/encoders/raw_rnn_encoder.py

```diff
@@ -24,8 +47,8 @@ class RawRNNEncoder(ModelPart, Attentive):
     def __init__(self,
                  name: str,
                  data_id: str,
-                 rnn_size: int,
-                 input_dimension: int,
+                 input_size: int,
+                 rnn_layers: List[Tuple],
```

Comment:
tady ten `assert check_argument_types()` o kousek níž ty typy kontroluje za runtimu a tohle je věc, která se dostává z konfigurace, tj. je potenciálně chybná. U jinejch metod to neni až tak důležitý.